ToDoApp/ToDoApp/Assets.xcassets/primaryButton.colorset/Contents.json

```diff
@@ -0,0 +1,20 @@
+{
+  "colors" : [
```

Comment:
I haven't seen the primary button colour in Figma. which one is this?

Reply:
It's the color of the add button in Figma 
<img width="88" alt="Screen Shot 2022-12-05 at 2 25 01 PM" src="https://user-images.githubusercontent.com/88293742/205756004-69fb82fe-5066-4f32-ba65-5dab275fc570.png">